docs/serving/autoscaling.md

```diff
@@ -1,16 +1,17 @@
 ---
-title: "Configuring the Autoscaler"
+title: "Autoscaling "
```

Comment:
This content still seems almost completely task oriented and lacks conceptual information (ie the "when" and "why" to configure each thing - for example when to choose KPA vs HPA and why, what are the pros and cons, when is it necessary to implement your own (is there a threshold?), etc.). Which it fine because that aligns with the Nav work in #1052 (-> Need to add new "Overview about Scaling in Knative")

Perhaps for this topic we keep the title aligned with a task, ie "Configuring scaling" or "Configuring autoscaling"?
And then add the necessary/missing Overview conceptual content in a new topic (as proposed [here](https://docs.google.com/spreadsheets/d/1L3EAeTX7RRdZhcQZeNmxwN4DCncuImCXA5A1_KQvwSk/edit#gid=413198505&range=45:45))?